<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TBMyPhone</class>
 <widget class="QWidget" name="TBMyPhone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="home" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>610</height>
    </rect>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>540</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pbFresh">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbScreenShot">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>截屏</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <zorder>widget_3</zorder>
    <zorder>pbFresh</zorder>
    <zorder>pbScreenShot</zorder>
   </widget>
   <widget class="QWidget" name="installApps" native="true">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>180</y>
      <width>210</width>
      <height>130</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>管理预装软件</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="scores">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="subtitle">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>手机定期体检,保持最佳状态</string>
    </property>
   </widget>
   <widget class="ScreenshotWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>230</width>
      <height>460</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pbConnectPhone">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>手机未连接</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="apps" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>210</width>
      <height>130</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>已装软件</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="files" native="true">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>315</y>
      <width>210</width>
      <height>130</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>文件</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="myPhoto" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>315</y>
      <width>210</width>
      <height>130</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>我的照片</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>270</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>手机上次体检得分:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>180</y>
      <width>240</width>
      <height>266</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbTest">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>80</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>立即体检</string>
    </property>
   </widget>
  </widget>
  <widget class="tbMyPhoneManager" name="next" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>610</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScreenshotWidget</class>
   <extends>QWidget</extends>
   <header location="global">screenshotwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tbMyPhoneManager</class>
   <extends>QWidget</extends>
   <header location="global">tbmyphonemanager.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
